.env

```
@@ -0,0 +1,5 @@
+WEB_PORT=8888
+INGEST_PORT=8084
+# WEBSOCKET_HOST could be your local IP (for development) or Public IP/hostname (for production) (yourdomain.tld)
+WEBSOCKET_HOST=192.168.156.180
+WEBSOCKET_PORT=8080
```

nitting, but a trailing empty line is nice 😄 

```suggestion
WEBSOCKET_PORT=8080

```